packages/documentation/gatsby-config.js

```diff
@@ -86,5 +66,51 @@ module.exports = {
     `gatsby-transformer-remark`,
     `github-api-pages`,
     `mdx-pages`,
+    {
+      resolve: `@gatsby-contrib/gatsby-plugin-elasticlunr-search`,
+      options: {
+        // Fields to index
+        fields: [`title`, `tags`],
+        // How to resolve each field`s value for a supported node type
+        resolvers: {
+          // For any node of type MarkdownRemark, list how to resolve the fields` values
+          SitePage: {
+            title: node => {
+              if (node.context && node.context.frontmatter && node.context.frontmatter.title) {
+                return node.context.frontmatter.title;
+              }
+
+              return '';
+            },
+            tags: node => {
+              if (node.context && node.context.frontmatter && node.context.frontmatter.tags) {
+                return node.context.frontmatter.tags;
+              }
+
+              return '';
+            },
+            path: node => node.path,
+          },
+          Mdx: {
+            title: node => {
+              console.log(node);
+              if (node && node.frontmatter && node.frontmatter.title) {
+                return node.frontmatter.title;
+              }
+
+              return '';
+            },
+            tags: node => {
+              if (node && node.frontmatter && node.frontmatter.tags) {
+                return node.frontmatter.tags;
+              }
+
+              return '';
+            },
+            path: node => `/visual-design/components${node.fields.slug}`,
```

Comment:
not a fan of this- I didn't see anything on the node I could pull off to get the path. maybe there's some graphQLing I could do to get the path?

Reply:
There's a query in SidebarItems for getting the list of component page paths for the sidebar. Maybe that would help?